### DISCUSSION TITLE
Support multiple selectors in "important" configuration option

### DISCUSSION THREAD

**Discussion by silvenon** [tailwindlabs/tailwindcss#10985] [Help] [answered]
I'm using Headless UI, which creates a React portal, and I also use Tailwind classes there. Would it be ok if Tailwind's `important` configuration option supported multiple selectors?

```js
module.exports {
  important: ['#app', '#headlessui-portal-root']
}
```

One thing to note here is that I'm using `important` to scope styles, rather than increase selector specificity. If there's another way to scope the same utility classes using multiple containers, I'm all ears!

**Comment by wongjn** [comment, marked as answer]
You could use `:is()`, like:

```js
module.exports {
  important: ':is(#app, #headlessui-portal-root)',
};
```

**Reply by silvenon** [reply]
That's genius, thank you!!! 😍

**Reply by KerryBlueGuru** [reply]
YEs!! Thank you so much for this!
